zeton/data_access/points.py

```diff
@@ -17,9 +17,9 @@ def change_points_by(target_id, points, user_id):
     get_db().commit()
 
 
-def add_exp(exp, child_id):
-    query = 'UPDATE main_points SET exp = exp + ?  WHERE child_id = ?;'
-    get_db().execute(query, [exp, child_id])
+def add_exp(exp, child_id, ex_id):
```

Comment:
i tak samo tu: nie wiem co to jest `exp`, czy to to samo co `experience`? Tylko z tym mi się kojarzy ;p

Reply:
Tu chodzi o experience tak jak mówisz :) jednak to nie jest to co zmieniałem w tym commicie